### DISCUSSION TITLE
关于 异常反馈

### DISCUSSION THREAD

**Discussion by pjy612** [pjy612/SteamManifestCache#5] [Question / Answer]
### 反馈类型
* 游戏长久未更新
  这种可以 先查一下 是否是 `miss` 标记的他人分享，一般更新时间为 每天晚上。
* 游戏配置异常 导致SAI入库操作后 解锁程序无法正常工作
  比如
  + 语言包缺失
  + 解锁项目或DLC 和 SteamDb 列表不匹配
* SAI 应用异常
  入库时出现明显的错误信息，麻烦附图

要额外提问的话，请先阅读
[提问的智慧](https://github.com/pjy612/How-To-Ask-Questions-The-Smart-Way/blob/main/README-zh_CN.md)  
[别像弱智一样提问](https://github.com/pjy612/Stop-Ask-Questions-The-Stupid-Ways/blob/master/README.md#%E5%88%AB%E5%83%8F%E5%BC%B1%E6%99%BA%E4%B8%80%E6%A0%B7%E6%8F%90%E9%97%AE)

**Comment by XiaoQ-xwx** [comment]
作者您好，请问一下只狼解锁后开启游戏弹出Application load error 6:0000065432的错误，请问是什么原因owo(是发这里吗？如果不是那对不起qvq)

**Reply by pjy612** [reply]
https://github.com/pjy612/SteamManifestCache/wiki/Steamless

**Comment by Bluse-liu50** [comment]
您好，游戏GINKA ID:2536840入库后可以下载，下载到的大小和steamdb上显示的应下载大小一样，启动时提示错误代码54，使用大号小号离线在线测试均有该问题。试图寻找方法排障无果，使用steamtools1.8最新beta，Steam 版本:  1700272093。

**Reply by pjy612** [reply]
SAI 如果 能在配合解锁工具后 让 Steam 显示 开始游戏
或者 能让 Steam 下载游戏内容。
就表示 SAI 功能正常。

具体游戏是否能玩，这个得依靠 解锁工具 如 tools或GL。 SAI 不负责 游玩。

**Comment by mengqian-666** [comment]
@pjy612 
![Snipaste_2023-12-04_20-34-25](https://github.com/pjy612/SteamManifestCache/assets/118544487/3acba46f-825a-4621-beb3-a2b00e3578bd)
废墟图书馆清单失效

**Reply by mengqian-666** [reply]
清单失效了qwq

**Reply by pjy612** [reply]
请自行比对 steamdb，你是不是自己额外解锁了DLC？目前没DLC清单。

**Reply by pjy612** [reply]
DLC清单找了下 已录入

**Reply by mengqian-666** [reply]
ok

**Comment by pjy612** [comment]
appid 1527950
缺DLC配置 2692400
